Trippy/View Controllers/Itinerary Tab/EditingItineraryViewController.m

```diff
@@ -31,11 +31,13 @@ @interface EditingItineraryViewController () <UITableViewDelegate, UITableViewDa
 @property (weak, nonatomic) IBOutlet UIView *editView;
 @property (weak, nonatomic) IBOutlet UITableView *placesTableView;
 @property (weak, nonatomic) IBOutlet UIActivityIndicatorView *loadingIndicator;
+@property (weak, nonatomic) IBOutlet UILabel *editArrow;
+@property BOOL viewIsOffscreen;
```

Comment:
I'm not sure what the defaults are for this - I think it would be good to make it explicit anyways, for clarity's sake. Also, nit: move this out of the `IBOutlet`s and down into the state variables below.